projects/aca-folder-rules/package.json

```diff
@@ -5,10 +5,10 @@
   "peerDependencies": {
     "@angular/common": "^14.1.0",
     "@angular/core": "^14.1.0",
-    "@alfresco/adf-core": "6.1.0-4944969415",
-    "@alfresco/adf-content-services": "6.1.0-4944969415",
-    "@alfresco/adf-extensions": "6.1.0-4944969415",
-    "@alfresco/js-api": "6.0.0",
+    "@alfresco/adf-core": "6.1.0-4985617493",
+    "@alfresco/adf-content-services": "6.1.0-4985617493",
+    "@alfresco/adf-extensions": "6.1.0-4985617493",
+    "@alfresco/js-api": ">=6.0.0",
```

Comment:
just to ensure - in some package.json there is now 6.1.0.925 in others >=6.0.0 for js-api. Is that expected?

Reply:
Yes for now we want to accept version higher than 6.0.0 in libs